api/src/main/resources/jakarta/persistence/orm_3_0.xsd

```diff
@@ -1057,14 +1057,15 @@
     <xsd:annotation>
       <xsd:documentation>
 
-        public enum GenerationType { TABLE, SEQUENCE, IDENTITY, AUTO };
+        public enum GenerationType { TABLE, SEQUENCE, IDENTITY, UUID, AUTO };
 
       </xsd:documentation>
     </xsd:annotation>
     <xsd:restriction base="xsd:token">
       <xsd:enumeration value="TABLE"/>
       <xsd:enumeration value="SEQUENCE"/>
       <xsd:enumeration value="IDENTITY"/>
+      <xsd:enumeration value="UUID"/>
```

Comment:
I guess this needs a new schema since this is JPA 3.1

Reply:
Yes, new schema is required. 2.2/3.0 (and all other already released versions) should be taken as written in the stone with no modifications allowed

Reply:
OK, I wasn't sure.

Reply:
@gavinking can you address this, please?